report-items/map/map-types/shape-map-series.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Shape Map Series
-page_title: Adding ShapeMapSeries Instances to the Map 
-description: "Learn how to add a ShapeMapSeries instance to the Map report item when using Telerik Reporting."
+page_title: Add Shape Map Series to the Map
```

Comment:
Try also wording this with initialize instead of add, for example:

Initializing a ShapeMapSeries Instance